livekit-plugins/livekit-plugins-google/livekit/plugins/google/beta/realtime/realtime_api.py

```diff
@@ -325,14 +326,48 @@ def fnc_ctx(self) -> llm.FunctionContext | None:
     def fnc_ctx(self, value: llm.FunctionContext | None) -> None:
         self._fnc_ctx = value
 
-    def _push_audio(self, frame: rtc.AudioFrame) -> None:
-        if self._opts.enable_user_audio_transcription:
-            self._transcriber._push_audio(frame)
+    def push_media_chunk(self, data: bytes, mime_type: str) -> None:
```

Comment:
```suggestion
    def _push_media_chunk(self, data: bytes, mime_type: str) -> None:
```

Reply:
@theomonnom the reason to keep this public is to give developers an ability to adjust the encoding method used (e.g. use PNG instead of JPEG, or a different size).

Reply:
just going to make it private for now and update the other comment so we can move this along